hawkbit-runtime/docker/docker-compose-pg.yml

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2018 Bosch Software Innovations GmbH and others.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+version: '3'
+
+services:
```

Comment:
I'm not sure if it still make sense to have another `docker-compose` file dedicated to PostgreSQL even if this setup can be run locally by activating the `postgresql` profile